app/packages/looker/src/lookers/abstract.ts

```diff
@@ -476,17 +470,13 @@ export abstract class AbstractLooker<
     }
 
     if (this.lookerElement.element.parentElement) {
-      const parent = this.lookerElement.element.parentElement;
-      this.resizeObserver.disconnect();
-      parent.removeChild(this.lookerElement.element);
-
-      for (const eventType in this.rootEvents) {
-        parent.removeEventListener(eventType, this.rootEvents[eventType]);
-      }
+      throw new Error("looker is attached");
```

Comment:
any situations where this is going to be a breaking change?

Reply:
Perhaps. Will change to `console.error`